crates/interpreter/src/interpreter.rs

```diff
@@ -86,7 +86,7 @@ impl Default for Interpreter<EthInterpreter> {
             InputsImpl {
                 target_address: Address::ZERO,
                 caller_address: Address::ZERO,
-                input: Bytes::default(),
+                input: CallInput::Bytes(Bytes::default()),
```

Comment:
We could hgave `CallInput::default()` that would create `CallInput::Bytes(Bytes::default())`